genet/output/geojson.py

```diff
@@ -209,3 +209,8 @@ def generate_standard_outputs(n, output_dir, gtfs_day='19700101', include_shp_fi
             output_dir=os.path.join(output_dir, 'routing'),
             include_shp_files=include_shp_files
         )
+
+    # import json
+    # summary_report = n.summary()
+    # with open(os.path.join(output_dir, 'summary_report.json'), 'w', encoding='utf-8') as f:
+    #     json.dump(sanitiser.sanitise_dictionary(summary_report), f, ensure_ascii=False, indent=4)
```

Comment:
delete?

Reply:
I actually needed to keep this, but I needed to fix the test with the old way of presenting attributes first (`{'attributes': {'osm:way:highway': 'text'}}`) - I have addressed the earlier comment now so it works with both notations now